cfpb/consumerfinance.gov
# Add raw HTML block to story pages

The content for our new story page involves some embedded charts and oversized headings. The raw HTML block will ensure we have enough control to make everything accessible if/when our standard blocks don't suffice.

## Additions

- Raw HTML block to story pages

## How to test this PR

1. Run migrations locally (or log into the PR preview website) and you'll see a raw HTML block when editing a story page.

## Screenshots

<img width="920" alt="Screen Shot 2021-05-27 at 9 44 37 PM" src="https://user-images.githubusercontent.com/1060248/119917544-c5a40f00-bf34-11eb-9f4e-e8e9a71b39de.png">

## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
